src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java

```diff
@@ -70,13 +75,24 @@ public boolean execute() {
         final Schema schema =
                 new Schema().id(schemaId).type(schemaType).schemaDefinition(definitionBase64).arguments(arguments);
 
+        final Schema createdSchema;
         try {
-            schemasApi.createSchema(schema);
+            createdSchema = schemasApi.createSchema(schema);
         } catch (final ApiException apiException) {
             outputFormatter.printApiException("Failed to create schema", apiException);
             return false;
         }
 
+        if (printEntireSchema) {
+            outputFormatter.printJson(createdSchema);
+        } else {
+            final JsonObject versionObject = new JsonObject();
+            if (createdSchema.getVersion() != null) {
+                versionObject.add(Schema.SERIALIZED_NAME_VERSION, new JsonPrimitive(createdSchema.getVersion()));
+            }
+            outputFormatter.printJson(versionObject);
+        }
```

Comment:
I am not sure if we should print the JSON response from the server for solely one create command. 
It seems a bit inconsistent with the policy create and the other delete commands.
I see three ways of how we could handle the ouput of REST API commands:
1. Never print the JSON response. Users would need to get the version via the get command. (I think this is the option chosen most widely)
2. Never print the JSON response and offer an opt-in option, as you did with `--print-result`, which can be combined with `--fields` to only get the version.
3. Always print the JSON response. (Stripe does this for example: https://stripe.com/docs/cli/delete)